packages/caver-klay/caver-klay-accounts/src/makeRawTransaction.js

```diff
@@ -177,7 +177,44 @@ function _combineFeePayerRawTransaction(rlpEncoded, sig, transaction, senderSign
   return txType + RLP.encode(rawTx).slice(2)
 }
 
+function extractSignatures(rawTransaction) {
+  let senderSignatures = []
+  let feePayerSignatures = []
+
+  const decoded = _decodeFromRawTransaction(rawTransaction)
+  senderSignatures = senderSignatures.concat(decoded.signatures)
+  if (decoded.feePayerSignatures) {
+    feePayerSignatures = feePayerSignatures.concat(decoded.feePayerSignatures)
+  }
+  return { senderSignatures, feePayerSignatures, decodedTransaction: decoded }
+}
+
+function decodeTxForFeePayer(rawTransaction) {
```

Comment:
What about changing this function name to `ClearFeePayerFields` or something like this?

Reply:
Umm ! how about makeFeePayerFormat?

Reply:
Is this making a fee payer format? I think it removes or clears the data in the fee payer fields.

Reply:
This function separates the fee payer signatures from the rawTransaction and creates the object in the fee payer transaction format for the fee payer to sign.

Since the fee payer's signatures are inside the returned object, wouldn't it be better to use an expression that creates a format for the fee payer rather than remove or clear?

Reply:
Then, what about using "split"?

Reply:
Oh, split sounds good to me !! i will change to that :)

Reply:
Name is changed to splitFeePayer !